test/integration/targets/ec2_vpc_vpn_facts/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- block:
+
+  # ============================================================
+  - name: test success with no parameters
+    ec2_vpc_vpn_facts:
+      region: '{{ ec2_region }}'
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+    register: result
+
+  - name: assert failure with no parameters
+    assert:
+      that:
+        - 'result.changed == false'
+        - 'result.vpn_connections == [] or result.vpn_connections != []'
+
+  # ============================================================
+  - name: test success with no parameters
```

Comment:
Could rename this since this test is using a parameter. These tests don't really test the output of the module, just that it can run. Ideally, there would be a VPN created to test vpn_connection_ids. Not a blocker though, more tests than plenty of facts modules have.